src/NServiceBus.Core/Transports/Msmq/ReceiveAndSendWithNativeTransaction.cs

```diff
@@ -6,9 +6,9 @@ namespace NServiceBus
     using System.Threading.Tasks;
     using Transport;
 
-    class ReceiveWithNativeTransaction : ReceiveStrategy
+    class ReceiveAndSendWithNativeTransaction : ReceiveStrategy
```

Comment:
ReceiveAndSend**s**WithNativeTransaction? I also like @bording his comment to use something more aligned with the transaction mode so it would become *SendsAtomicWithReceiveUsingNativeTransactions*